Added FAQ to Google API Key (#491)
appcircleio/appcircle-docusaurus: docs/account/my-organization/api-integrations/adding-google-play-service-account.md

Then click **Invite User**. Your account key is ready. 🎉

14. To add the key on Appcircle, follow these steps:

>     a. Navigate to [My Organization](/account/my-organization)n.

Reviewer: ```suggestion
    a. Navigate to [My Organization](/account/my-organization).
```

Reviewer: @ftmzhrsckldr I fixed this typo. Thnks for your effort. I approved

    b. Locate the `Google Play Developer API Keys` under the `Connections` section.
  
    c. Click the `Manage` button if you have saved keys, or directly click the `Add New` button.

<Screenshot url='https://cdn.appcircle.io/docs/assets/google-service14.png' />

## FAQ

**Can I retrieve the JSON private key that I uploaded to Appcircle?**

> No, for security reasons, the JSON file you uploaded cannot be downloaded again from Appcircle.

Reviewer 2: ```suggestion
No, for security reasons, you cannot download the JSON file you uploaded to Appcircle again.
```

> **After uploading the JSON, why can't I save it?**

Reviewer 2: ```suggestion
**Why can’t I save the JSON after uploading it?**
```

> You may have skipped the first step. Please ensure you have enabled the Service account.

Reviewer 2: ```suggestion
You might have missed the first step. Please ensure that you have enabled the Service Account.
```
